devfile.yaml

```diff
@@ -0,0 +1,40 @@
+schemaVersion: 2.2.0
```

Comment:
I've been using the Developer Sandbox to work with this repo, so thought I'd add a devfile in, incase others want to use Dev Spaces